docs/src/test-assertions-js.md

```diff
@@ -13,10 +13,10 @@ Playwright also extends it with convenience async matchers that will wait until
 the expected condition is met. Consider the following example:
 
 ```js
-await expect(page.locator('.status')).toHaveText('Submitted');
+await expect(page.getByTestId('status')).toHaveText('Submitted');
 ```
 
-Playwright Test will be re-testing the node with the selector `.status` until fetched Node has the `"Submitted"` text. It will be re-fetching the node and checking it over and over, until the condition is met or until the timeout is reached. You can either pass this timeout or configure it once via the [`property: TestConfig.expect`] value in test config.
+Playwright Test will be re-testing the node with the test id of `status` until the fetched Node has the `"Submitted"` text. It will re-fetch the node and check it over and over, until the condition is met or until the timeout is reached. You can either pass this timeout or configure it once via the [`property: TestConfig.expect`] value in the test config.
```

Comment:
```suggestion
Playwright Test will be re-testing the element with the test id of `status` until the fetched element has the `"Submitted"` text. It will re-fetch the element and check it over and over, until the condition is met or until the timeout is reached. You can either pass this timeout or configure it once via the [`property: TestConfig.expect`] value in the test config.
```